packages/components/src/internal/components/search/FilterValueDisplay.tsx

```diff
@@ -41,11 +41,11 @@ export const FilterValueDisplay: FC<FilterValueDisplayProps> = memo(props => {
     const { filter, onFilterValueExpand, noValueInQueryFilterMsg } = props;
 
     const exclude = useMemo(() => {
-        return NEGATE_FILTERS.indexOf(filter.getFilterType().getURLSuffix()) > -1;
+        return NEGATE_FILTERS.indexOf(filter?.getFilterType().getURLSuffix()) > -1;
     }, [filter]);
 
     const filterTypeLabel = useMemo(() => {
-        const filterType = filter.getFilterType();
+        const filterType = filter?.getFilterType();
```

Comment:
If `filter` ends up being `undefined` then `filterType` will be `undefined`. That said, I'd recommend reverting the changes to this file and not be defensive here against a null/undefined `filter`. You accounted for this in `FilterCards.tsx`.

Reply:
Done.